### PR TITLE
Added max element in stack

### LINKED ISSUE #5232: Maximum element
I would like to do these operations in stack folder using C++ :-

1. Push the element x into the stack.
2. Delete the element present at the top of the stack.
3. Print the maximum element in the stack.

### PR DESCRIPTION
**Pull request by sudipg4112001**
Fixes #5232

### CODE REVIEW

**Comment by sudipg4112001** [comment]
Made the changes, please do review.

File: C-Plus-Plus/stack/Max_element.cpp [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* Two stacks are kept: One is for pushing and popping elements as they come and go (elements stack), and the other is for keeping track of the maximum element (maximum elements stack).
+When the first element arrives, push it onto both of the stacks. As the elements may not be distinct, we need to keep track of their indices. So, declare a stack of pair<int,int> or use a structure.*/
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/stack/Max_element.cpp:2]: Make it short and readable

File: C-Plus-Plus/stack/Max_element.cpp [hunk 2]
```diff
@@ -0,0 +1,79 @@
+/* Two stacks are kept: One is for pushing and popping elements as they come and go (elements stack), and the other is for keeping track of the maximum element (maximum elements stack).
+When the first element arrives, push it onto both of the stacks. As the elements may not be distinct, we need to keep track of their indices. So, declare a stack of pair<int,int> or use a structure.*/
+#include <iostream>
+#include <vector>
+#include <cstring>
+#include <stack>
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/stack/Max_element.cpp:6]: Replace header with `#include <bits/stdc++.h>`

File: C-Plus-Plus/stack/Max_element.cpp [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        }
+        else
+        {
+            p1 = maxi.top();
+            cout<<p1.first<<endl;
+        }
+        i++;
+    }
+    return 0;
+}
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/stack/Max_element.cpp:55]: - Shift the logic to another function and use main for driver code
- Use proper variable naming conventions , Don't use short forms or single letters

File: C-Plus-Plus/stack/Max_element.cpp [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        3
+        1 91
+        3
+        
+ TEST CASE:2
+ INPUT: 4              OUTPUT: 83 
+        1 83
+        3
+        2
+        1 76
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/stack/Max_element.cpp:77]: Remove extra test case , one is enough also use proper display message while reading and displaying the outputs

File: C-Plus-Plus/stack/Max_element.cpp [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        {
+            p1 = maxi.top();
+            cout<<p1.first<<endl;
+        }
+        i++;
+    }
+    return 0;
+}
+
+/*  Time Complexicity: O(n)             Space Complexicity: O(n)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/stack/Max_element.cpp:57]: rewrite it as, 
```
Time Complexicity: O(n)             
Space Complexicity: O(n)
```

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/README.md [hunk 6]
```diff
@@ -312,6 +312,7 @@
 - [Sort stack using recursion](stack\Sort_stack_using_recursion.cpp)
 - [Checking if a linked list is palindromic](stack/palindromic_list.cpp)
 - [Check palindrome using stack](stack/CheckPalindrome.cpp)
+- [Maximum Element](C-Plus-Plus/stack/Max_element.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md:318]: Rewrite it as `[Maximum Element using Stack](stack/Max_element.cpp)`

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Kindly don't push from master branch, Closing this PR open a  new one and do the changes as I mentioned.

**Pull request closed without merging**